log4j-flume-ng/src/test/java/org/apache/logging/log4j/flume/appender/FlumeEmbeddedAppenderTest.java

```diff
@@ -208,7 +208,7 @@ public void testHeaderAddedByInterceptor() {
     }
 
     /* @Test */
-    public void testPerformance() throws Exception {
+    private void testPerformance() throws Exception {
```

Comment:
I'd rather keep these tests in with `@Test` annotation, though add a `@Disabled` with explanation. (There was also one in `PerformanceComparison`.)